src/PrestaShopBundle/Controller/Admin/CommonController.php

```diff
@@ -111,6 +112,14 @@ public function notificationsAckAction(Request $request)
      */
     public function paginationAction(Request $request, $limit = 10, $offset = 0, $total = 0, $view = 'full', $prefix = '')
     {
+        // Cf. https://github.com/PrestaShop/PrestaShop/pull/28782
```

Comment:
Not sure we need to keep the PR URL in the comment here, simply referencing the code itself (`DoctrineGridDataFactory::getData`) is enough for the explanation, or else we'll start having dome PR URLs on every change we make 😅

If someone really needs to find the initial PR he can always use `blame` to find the original modification